py/tracks/latest.toml

```diff
@@ -47,5 +47,8 @@ full = [
     "../specs/select/*.toml",
     "../specs/in_subquery.toml",
     "../specs/in_subquery.toml",
-    "../specs/union.toml"
+    "../specs/union.toml",
+    "../specs/select/group_by_no_doc_values.toml",
```

Comment:
I wonder if we should create a new track file and make use of the `fixtures`, `queries` functionality (to run the setup only once, for all 3 files)

The full  suite currently already runs +8hrs on jenkins.